codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -37,30 +39,28 @@
  * Typically this thread is the event loop thread for the {@link ChannelHandlerContext} managed by this class.
  */
 public class DefaultHttp2RemoteFlowController implements Http2RemoteFlowController {
+    private static final InternalLogger logger =
+            InternalLoggerFactory.getInstance(DefaultHttp2RemoteFlowController.class);
     private static final int MIN_WRITABLE_CHUNK = 32 * 1024;
-    private final Http2StreamVisitor WRITE_ALLOCATED_BYTES = new Http2StreamVisitor() {
-        @Override
-        public boolean visit(Http2Stream stream) {
-            int written = state(stream).writeAllocatedBytes();
-            if (written != -1 && listener != null) {
-                listener.streamWritten(stream, written);
-            }
-            return true;
-        }
-    };
     private final Http2Connection connection;
     private final Http2Connection.PropertyKey stateKey;
-    private int initialWindowSize = DEFAULT_WINDOW_SIZE;
-    private ChannelHandlerContext ctx;
-    private Listener listener;
+    int initialWindowSize = DEFAULT_WINDOW_SIZE;
```

Comment:
Still, exposing it as package private doesn't seem like a good idea.  If you were using it for tests that would be one thing, but making it package private just to avoid method generation seems like overkill.